package.json

```diff
@@ -59,6 +60,7 @@
         "lint-markdown:fix": "markdownlint -f --ignore node_modules '**/*.md'",
         "startrc": "node scripts/test-runner.js startrc",
         "check-code-samples": "node scripts/test-runner.js check-code-samples",
+        "check-markdown-broken": "markdown-link-check -q",
```

Comment:
I believe the name of the script can be improved. `check-markdown-links` or even `check-markdown` would be better.